sematic/ui/packages/common/src/pages/RunDetails/sourcecode/SourceCode.tsx

```diff
@@ -0,0 +1,30 @@
+import Box from "@mui/material/Box";
+import { Light as SyntaxHighlighter } from "react-syntax-highlighter";
+import docco from "react-syntax-highlighter/dist/esm/styles/hljs/docco";
+import python from "react-syntax-highlighter/dist/esm/languages/hljs/python";
+import { Run } from "src/Models";
+
+SyntaxHighlighter.registerLanguage("python", python);
+
+interface SourceCodeProps {
+    run: Run;
+}
+
+function SourceCode(props: SourceCodeProps) {
+    const { run } = props;
+  
+    return (
+        <Box key={run.function_path} sx={{ marginTop: 2 }}>
```

Comment:
Can we remove the left and right margin so that there is no white space between the edge and the source.
Instead of a top margin, can you use a top padding so that there is no white space?
Also can we make the background of the whole tab of the same color as the source code area?

Happy to discuss this together later today.

Reply:
Of course. Now it looks like this:

<img width="1586" alt="image" src="https://github.com/sematic-ai/sematic/assets/133257643/b935c77e-7154-4e67-a64d-fd76ab9987e8">